modules/expo-apple-ads/src/index.ts

```
@@ -0,0 +1,35 @@
+import { requireNativeModule } from 'expo-modules-core';
```

ESLint

[error] Delete `;`

(prettier/prettier)